.github/workflows/test-performance-weekly.yml

```
@@ -0,0 +1,43 @@
+  schedule:
+    - cron: '0 5 * * 1'
+  workflow_dispatch:
+
+permissions:
+  id-token: write
+  contents: read
+
+run-name: "Weekly Performance ${{ github.ref_name }} #${{ github.run_number }} ${{ github.sha }}"
+
+jobs:
+  create-and-upload 20/15m/1:
```

actionlint (1.7.7)

invalid job ID "create-and-upload 20/15m/1". job ID must start with a letter or _ and contain only alphanumeric characters, -, or _

(id)